lib/gis/lrand48.c

```diff
@@ -72,6 +72,7 @@ long G_srand48_auto(void)
 {
     unsigned long seed;
     char *grass_random_seed = getenv("GRASS_RANDOM_SEED");
+    grass_random_seed ||= getenv("SOURCE_DATE_EPOCH");
```

Comment:
This leads in Travis to:

```
gcc  -Werror=implicit-function-declaration  -fPIC  -I/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/include -I/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/include   -DGRASS_VERSION_DATE=\"'2019'\" -DPACKAGE=\""grasslibs"\"       -I/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/include -I/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/include -DRELDIR=\"lib/gis\" -o OBJ.x86_64-pc-linux-gnu/mapset.o -c mapset.c
lrand48.c: In function ‘G_srand48_auto’:
lrand48.c:75:25: error: expected expression before ‘=’ token
     grass_random_seed ||= getenv("SOURCE_DATE_EPOCH");
                         ^
../../include/Make/Compile.make:32: recipe for target 'OBJ.x86_64-pc-linux-gnu/lrand48.o' failed
make[3]: *** [OBJ.x86_64-pc-linux-gnu/lrand48.o] Error 1
```

Reply:
oops. It seems that notation only works in perl, but it looked so C-like. Rewrote it now.